SA/design_concept/content/results/results_requirement_validation.typ

```diff
@@ -52,5 +52,5 @@
 The project's main focus was to create a visual concept for function composition and prove that it is feasible with a #ac("PoC") application.
 As can be seen in @requirement_validation_table, all requirements related to function composition have been achieved.
 
-Due to time constraints, the requirements #ac("UC")2, #ac("UC")7, #ac("UC")8 and #ac("UC")9 had to be considered as out-of-scope.
+Due to time constraints, the requirements #ac("UC") 2, #ac("UC")7, #ac("UC")8 and #ac("UC")9 had to be considered as out-of-scope.
```

Comment:
Was this by intention? Otherwise, we should add a space to the other references as well

Reply:
This is the first occurence of the UC abbreviation, so this displays `UC (Use Case)2` without the additional space.

Reply:
hmm okey I see. Not I'm not soooo happy about that abbreviation feature anymore... But it does make sense 👍

Reply:
Should we in this case maybe remove the acronyms here and use identifiers only?

Reply:
> hmm okey I see. Not I'm not soooo happy about that abbreviation feature anymore... But it does make sense 👍

Yeah, I have mixed feelings about it. For most things it makes sense, but in some cases, like this or NFRs, it causes some problems. I'll remove the acronyms for this case.